nav_msgs/srv/LoadMap.srv

```diff
@@ -0,0 +1,3 @@
+string map_path
```

Comment:
I just realized [SetMap](https://github.com/ros/common_msgs/blob/jade-devel/nav_msgs/srv/SetMap.srv) already exists, so that's one less service to define.